trestle/core/commands/author/ssp.py

```diff
@@ -498,6 +498,12 @@ def _run(self, args: argparse.Namespace) -> int:
 
                 new_file_content_type = FileContentType.path_to_content_type(orig_ssp_path)
             else:
+                if not args.compdefs:
+                    logger.warning(
+                        'No comp_defs have been specified and a destination SSP does not exist.  If you want the full '
+                        'set of components in the SSP you should specify the needed comp_def names.  The SSP '
+                        'will now attempt to be assembled without them but may fail.'
+                    )
```

Comment:
Got curious and just tested this scenario (no comp_def specified and no previously assembled SSP), this will actually fail with very general 'KeyError' error. Maybe it is better to not only give a warning but modify the code so this scenario is handled correctly.

Reply:
Removed this warning and will leave associated issue open.